integration/pxe/Dockerfile.sztp

```diff
@@ -1,2 +1,9 @@
 FROM python:3.9
 RUN pip install sztpd==0.0.11
+
+# for simulator
+RUN apt update && apt install -y libyang-tools libxml2-utils
+# FIXME: why ADD is not working here?
```

Comment:
What isn't working with ADD?  Usually when I have problems with ADD it's because the thing I'm adding isn't in the build context.